extensions/replication/ReplicationConfigValidator.js

```diff
@@ -36,22 +51,22 @@ const joiSchema = joi.object({
             }).when('type', { is: 'role', then: joi.required() }),
         }).required(),
     },
-    destination: {
+    destination: joi.object({
         transport: transportJoi,
-        auth: joi.object({
-            type: joi.alternatives().try('account', 'role', 'service')
-                .required(),
-            account: joi.string()
-                .when('type', { is: 'account', then: joi.required() }),
-            vault: joi.object({
-                host: joi.string().optional(),
-                port: joi.number().greater(0).optional(),
-                adminPort: joi.number().greater(0).optional(),
-                adminCredentialsFile: joi.string().optional(),
-            }),
-        }).required(),
+        auth: destinationAuthJoi.optional(),
+        sites: joi.object().pattern(
+            joi.string(), // site name
+            joi.object({
+                transport: transportJoi,
+                auth: destinationAuthJoi.required(),
+            }).required()
+        ).when('auth', {
```

Comment:
should this condition on be on each destination's `auth` field ?
(i.e. the site's auth must be specified if there is no "top level" default)

Reply:
This is already validated in `_validatePerSiteDestinationConfig`